slash/plugins/builtin/email_template.j2

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml" xmlns:v="urn:schemas-microsoft-com:vml" xmlns:o="urn:schemas-microsoft-com:office:office">
+<head>
+    <meta charset="utf-8"> <!-- utf-8 works for most cases -->
+    <meta name="viewport" content="width=device-width"> <!-- Forcing initial-scale shouldn't be necessary -->
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"> <!-- Use the latest (edge) version of IE rendering engine -->
+    <meta name="x-apple-disable-message-reformatting">  <!-- Disable auto-scale in iOS 10 Mail entirely -->
+    <title>{{title}}</title> 
+</head>
+<body width="100%" bgcolor="silver">
+  <h1><img src="http://slash.readthedocs.org/en/latest/_static/slash-logo.png" style="height: 2.2em; margin-right: 1em; position: relative; top: 16px">{{title}}</h1>
+  <p>
+    Session <a href="{{backslash_link}}">{{session_id}}</a>, running on <strong>{{host_name}}</strong>  has finished {{result}} after {{duration}}.
+  </p>
```

Comment:
In case baskslash plugin is not active - there won't be `backslash_link`. 
Should we set it to empty string in `kwargs` or should we handle it here?

This bug is originated in previous commit which used `_HTML_TEMPLATE` (sorry - my mistake).